en/docs/insights/programmatic-access-choreo-insights-api.md

```diff
@@ -0,0 +1,62 @@
+# Programmatic Access to Insights API
+
+The **Insights** page in the Choreo Console fetches data from the Insights API exposed over the Internet. However, in some scenarios, you may need to fetch data from external systems and implement custom dashboards to display this data.
+
+Choreo provides a secure and user-friendly method to achieve this requirement. The steps are outlined below.
+
+## Become a member of the Choreo System organization
+
+1. Sign in to [https://devportal.choreo.dev](https://devportal.choreo.dev) using your credentials.
+
+2. Access the developer portal of the `Choreo System` organization via [https://devportal.choreo.dev/choreosystem](https://devportal.choreo.dev/choreosystem).
+   
+3. Request Access to the `Choreo System` organization by clicking **Request Access**.
```

Comment:
Unnecessary capitalization on the term access.

Shall we say
"Click **Request Access** to get access to the `Choreo System` organization."
or 
"To get access to the `Choreo System` organization, click **Request Access**.

Reply:
Fixed with [54aec2f](https://github.com/wso2/docs-choreo-dev/pull/437/commits/54aec2f020296f46f683ab402e4183683e433051)